com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/IPluginSubTargetMaterialUtils.cs

```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+
+namespace UnityEditor.Rendering.HighDefinition.ShaderGraph
+{
+    public static class PluginMaterial
+    {
+        public enum GenericVersions
+        {
+            NeverMigrated = 0,
+            Initial = 1,
+        }
+        public const int k_NeverMigratedVersion = 0;
+    }
+    public interface IPluginSubTargetMaterialUtils
+    {
+        int latestMaterialVersion { get; }
+        int latestSubTargetVersion { get; }
```

Comment:
This is again to support the MaterialPostProcessor in reimporting a shadegraph on which a material depends.